projects/aca-folder-rules/src/lib/rule-details/actions/rule-action.ui-component.ts

```diff
@@ -194,7 +194,7 @@ export class RuleActionUiComponent implements ControlValueAccessor, OnInit, OnDe
             return new CardViewTextItemModel({
               ...cardViewPropertiesModel,
               icon: 'folder',
-              default: this.translate.instant('ACA_FOLDER_RULES.RULE_DETAILS.PLACEHOLDER.CHOOSE_FOLDER'),
+              default: '',
```

Comment:
I can see that this change changes look, see screenshot: 
<img width="1658" alt="image" src="https://user-images.githubusercontent.com/115619721/225232442-b6e63b26-667a-415e-99e5-4d0a93b23c0a.png">

I can see in destination folder there is "Choose destination folder" placeholder. I guess it will be lost after that change? Can we do that type of change or should we ask somebody from UI designers if that is fine? @TomikaArome @nikita-web-ua @MichalKinas

Reply:
If you have this default value as placeholder, it is considering this as actual value & that's why 'create' or 'update' button is getting enabled without selecting the file. If you see others controls like text-box are also without placeholder. There was no placeholder for any controls except file upload.

Reply:
ok then I guess should be fine